PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# PULL REQUEST TEMPLATE
+
+## Description
+
+Please provide a brief description of the changes made in this pull request.
+
+## Related Issue
+
+If this pull request is related to any GitHub issue, please mention it here with the appropriate issue number.
+
+## Proposed Changes
+
+List down the specific changes or features introduced in this pull request.
+
+- Feature/issue-number/feature-description: [Feature Description]
+- Refactor/issue-number/issue-description: [Issue Description]
+- ...
+
+## Additional Information
+
+If there's any additional information or context that the reviewers should know, please provide it here.
+
+## Checklist
+
+Please check the following before submitting your pull request:
+
+- [ ] I have tested my changes locally and they are working as expected.
+- [ ] I have added necessary documentation or updated existing documentation.
+- [ ] I have added relevant tests for the proposed changes.
+- [ ] My code follows the project's coding guidelines and style.
+- [ ] I have reviewed my own code and ensured its quality.
+
+## Reviewers
+
+Tag the relevant project maintainers or team members who should review this pull request.
+
+@agitrubard
+@skayikci
+@Rapter1990
```

Comment:
Reviewers kısmı otomatik olarak setlendiği için bence belirtmemize gerek yok, sadece default reviewer'ların olması şartını belirtelim, kim olduğunu burada belirtmeyelim.

Reply:
I think this sentence can be enough to explain

`The default project maintainers and team members will be automatically assigned as reviewers for this pull request.`